sdk/src/main/scala/com/horizen/account/state/AccountState.scala

```diff
@@ -191,7 +198,11 @@ class AccountState(val params: NetworkParams,
 
   // View
   override def getView: AccountStateView = {
-    new AccountStateView(stateMetadataStorage.getView, messageProcessors)
+    // get state root
+    val stateRoot = stateMetadataStorage.getAccountStateRoot.getOrElse(new Array[Byte](32))
+    val statedb = new StateDB(stateDbStorage, stateRoot)
```

Comment:
How do we make sure that `statedb.close()` is properly called for each instance of `StateDB`? We will have memory leaks otherwise. The same is true for every instance of Database.

Reply:
Added a close() method to the AccountStateView class and I also extended it with AutoCloseable.